src/support/azext_support/_validators.py

```diff
@@ -134,10 +135,31 @@ def _check_name_availability_no_subscription(cli_ctx, resource_name, resource_ty
     if not resp["nameAvailable"]:
         raise CLIError(resp["message"])
 
-
 def _check_name_availability_no_subscription_ticket(cli_ctx, ticket_name, resource_name, resource_type):
     from .aaz.latest.support.no_subscription.tickets import CheckNameAvailability
     check_name_availability_input = {"support_ticket_name": ticket_name, "name": resource_name, "type": resource_type}
     resp = CheckNameAvailability(cli_ctx=cli_ctx)(command_args=check_name_availability_input)
     if not resp["nameAvailable"]:
         raise CLIError(resp["message"])
+
+def validate_file_path(file_path):
+    if not os.path.exists(file_path):
+        raise CLIError("File does not exist!")
+    
+def validate_file_name(file_name):
+    max_file_name_length = 110
+    if len(file_name) > max_file_name_length:
+        raise CLIError("File name should not be more than 110 characters.")
+    
+def validate_file_size(file_size):
+    max_file_size = 5242880
```

Comment:
Turn this into constant variable values at the top

Reply:
Done